eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -255,6 +255,12 @@
           "new": "class com.azure.messaging.eventhubs.EventData",
           "superClass": "com.azure.core.models.MessageContent",
           "justification": "To support Schema Registry Avro Serializer's signature, EventData extends from MessageContent."
+        },
+        {
+          "regex": true,
+          "code": "java.class.externalClassExposedInAPI",
+          "new": "class com\\.azure\\.messaging\\.eventhubs\\..*",
+          "justification": "SDK classes are allowed to be exposed by dependencies using them."
```

Comment:
Why is this required?

Reply:
Without adding this in, the project was not being built. The error that was shared was "A class from supplementary archives is used in a public capacity in the API", to address that we exposed the APIs from Jedis. Is there any alternative way to approach to fix that error?

Reply:
@srnagar let's file a follow up issue to further discuss when exposing APIs from other libraries should be suppressed. Right now, we suppress Core APIs, Jackson, Reactor, and a few other external dependencies, and this looks like we need to be broader when it comes to exposing APIs from other Azure SDKs.